src/main/java/core/utils/DateFormatter.java

```diff
@@ -0,0 +1,12 @@
+package core.utils;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DateFormatter {
+    public static LocalDate dateFormatter(String dateInString) {
```

Comment:
For me this class seems over-engineering. you may easily use 
private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
in your class and use it as argument in code